wmn-data.json

```diff
@@ -5307,7 +5307,7 @@
        },
        {
         "name" : "Taringa Archived Profile",
-        "uri_check" : "http://archive.org/wayback/available?url=taringa.net/{account}",
+        "uri_check" : "http://archive.org/wayback/available?url=http://www.taringa.net/{account}",
         "uri_pretty" : "https://web.archive.org/web/2/taringa.net/{account}",
         "e_code" : 200,

```

Comment:
When I visit `https://web.archive.org/web/2/taringa.net/farantic` it redirects my browser (HTTP response code 302) to `https://web.archive.org/web/20201230200907/https://www.taringa.net/Farantic`. Same thing for the other known account `https://web.archive.org/web/20130526001955/http://www.taringa.net/Elrubius`. In this case, I think that is a good thing. No changes needed.